### PR TITLE
Add edge and track angle features

### LINKED ISSUE #4: Add new track feature: angle
Hi @tinevez,

I was looking for a track feature measuring the directionality/angle of an entire track (i.e. the line between start and end; or alternatively some weighted mean of the single edge angles??).

I couldn't find anything like that, but I think this repository here would be suitable to hold such a track feature.

Are you accepting pull requests here, or should I rather implement this in my own add-on (i.e. it would end up in https://github.com/fmi-faim/fmi-trackmate-addons/)?

If there's some more suitable place for contributions like this, e.g. in Mastodon, I'm all ears 😉

### PR DESCRIPTION
**Pull request by imagejan**
Closes #4.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -82,6 +82,9 @@
 		<license.copyrightOwners>Fiji developers.</license.copyrightOwners>
 		<license.projectName>Fiji distribution of ImageJ for the life sciences.</license.projectName>
 
+		<!-- https://github.com/scijava/scijava-coding-style -->
+		<scijava.coding-style>imglib2</scijava.coding-style>
```
Comment (tinevez) [inline pom.xml:86]: What this line does? Is it black magic?

Comment (imagejan) [reply pom.xml:86]: That's a recent addition by @maarzt to the SciJava ecosystem, shipping with the latest `pom-scijava`.

You can run:

```
mvn formatter:format
```

and

```
mvn impsort:sort
```

to get your code formatted and imports sorted, in the style of your choice.
See the link in the code.

**Comment by imagejan** [comment]
Note:

This pull request implements functionality similar to [`EdgeAngleAnalyzer`](https://github.com/fiji/TrackMate-examples/blob/master/src/main/java/plugin/trackmate/examples/edgeanalyzer/EdgeAngleAnalyzer.java) in `TrackMate-examples`.

In addition to the 2d edge angle features in the linked class, it adds:

* 3d edge angles (xy, yz, zx)
* Angles for entire tracks

**Comment by tinevez** [comment]
Thanks Jan!

**Pull request merged**